baqend/_ConfigBuilder.ts

```diff
@@ -7,9 +7,16 @@ export class ConfigBuilder {
   private whitelist: Rule[] = []
   private blacklist: Rule[] = []
 
-  constructor(appName: string, userAgentDetection: boolean) {
+  /**
+   * The current size of the config counting all included conditions.
+   */
+  private size: number = 0
+  private maxCapacity: number
+
+  constructor(appName: string, userAgentDetection: boolean, maxCapacity: number = -1) {
```

Comment:
You should put `CONFIG_MAX_SIZE` (see below) in __ConfigBuilder.ts_, export it, and use it as `maxCapacity`'s default:
```js
constructor(appName: string, userAgentDetection: boolean, maxCapacity: number = CONFIG_MAX_SIZE) {
```